EIP 4844 - blobhash opcode (part 1) (#14757)
ethereum/solidity: Changelog.md

### 0.8.24 (unreleased)

Language Features:

>  * Yul: Introduce builtin ``blobhash()`` for retrieving a versioned blob hash of a given blob index within the transaction.

Reviewer: How about this wording?
```suggestion
 * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.
```

### 0.8.24 (unreleased)

Language Features:

>  * Yul: Introduce builtin ``blobhash()`` for retrieving a versioned blob hash of a given blob index within the transaction.

Reviewer: ```suggestion
 * Yul: Introduce builtin ``blobhash()`` for retrieving a versioned blob hash of a given blob index within the transaction.

```

Language Features:
 * Introduce global ``block.blobbasefee`` for retrieving the blob base fee of the current block.
 * Yul: Introduce builtin ``blobbasefee()`` for retrieving the blob base fee of the current block.

>  * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.

Reviewer: Oh, one more thing, let's finally stop messing with the spacing in the changelog :)

```suggestion
 * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.
 
```

I fixed it earlier in all those PRs but it keeps reappearing again and again for some reason. Don't worry about the wrong spacing of the `Compiler Features:` section. We'll fix it when we sort the changelog. In PRs just add an entry for your PR and don't touch other lines.
